docs/tutorial/offscreen-rendering.md

```diff
@@ -57,3 +57,16 @@ app.once('ready', () => {
 ```
 
 [disablehardwareacceleration]: ../api/app.md#appdisablehardwareacceleration
+
+## User Input
+
+When a window is rendered offscreen, Electron has no way to route user input to
+your window. This means that if you want to have user input in your
+application that is being rendered offscreen, you need to provide it yourself.
+
+You can send input events via the [`webContent.sendInputEvent()`]( https://electronjs.org/docs/api/web-contents#contentssendinputeventevent)
+method in `webContent`. You should also be sure to focus the window via the
+[`win.focusOnWebView()`](https://electronjs.org/docs/api/browser-window#winfocusonwebview)
```

Comment:
This seems more like a bug than expected behavior, I feel as though we should fix `.focus` rather than rely on a side effect of another API

Reply:
Yeah, not a fan of documenting a "workaround", let's give that issue some time to be reviewed and potentially fixed before add documentation that would imply this is "expected" behavior.